sphinx_needs/directives/needimport.py

```diff
@@ -92,8 +92,11 @@ def run(self) -> Sequence[nodes.Node]:
             )[1]
 
             if not os.path.exists(correct_need_import_path):
-                raise ReferenceError(
-                    f"Could not load needs import file {correct_need_import_path}"
+                log_warning(
```

Comment:
This will still raise a `FileNotFoundError` exception below (caught by `OSError`). There is also a TODO to add exception handling below. I think we should return a simple docutils paragraph node that warns the reader about the about missing needimport. And then, as you do, emit a warning.